color_refinement.py

```diff
@@ -221,18 +221,21 @@ def get_number_automorphisms(g: Graph) -> int:
 
     graphs = L[0]
     print("Graph: ", GRAPH)
+    automorphisms = {}
     for i in range(len(graphs)):
         for j in range(len(graphs)):
             if j == i:
                 start = time.time()
                 num = get_number_automorphisms(graphs[i])
-                print('There are', num, 'automorphisms')
+                automorphisms[graphs[i].name] = num
```

Comment:
In #53 gebruik ik ook dictionaries en gebruik ik de index van de graaf als key. De naam van een graaf vind ik ook prima om als key te gebruiken, maar is die altijd uniek?

Reply:
Ik had de graph_io getweaked zodat de graaf naam van graphs[i] "G"+str(i) is.

Reply:
Aangezien we in deze loopjes indexeren met `i` en `j`, denk ik toch dat `automorphisms` het beste ook met `i` en `j` als keys kan worden gebruikt.